jaxb/pom.xml

```diff
@@ -97,6 +97,40 @@ data-binding.
       <plugin>
         <groupId>org.moditect</groupId>
         <artifactId>moditect-maven-plugin</artifactId>
+        <inherited>true</inherited>
+        <configuration>
+          <jvmVersion>11</jvmVersion>
```

Comment:
this solves the javassist and jaxen issues in older jdk 7 frameworks reading module-info files, as well as doesn't get removed during the shade for jakarta namespacing

this was the trick to make it all work

Reply:
Do we care about JDKs 9 and 10? They are not LTS versions but just in case...

Reply:
There is always that one person so it definitely is safer - 

I double checked - for me I put it there as other modules override at 11, and i needed to overwrite those xD So no issue there will update